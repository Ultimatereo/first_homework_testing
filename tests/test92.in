5
-677889066170 -73714925521 402457913438 262315792196 -340357961971 -317868113642
146767571286 980177296525 -407179232142 -937364773265 -147088937767 -901652203800
-227654251084 265426219694 583314809508 144900496083 -335541041215 187615631234
208621435395 996634930028 259718234605 -668765586414 906264303029 30954108384
-161371444220 -906219718777 902673841872 -341773459249 -201909145261 923113226270
