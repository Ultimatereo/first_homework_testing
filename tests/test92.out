1.11191
-0.401968
0.956268
0.276685
0.150373
